TF2 Rich Presence/build.py

```diff
@@ -477,7 +477,7 @@ def copy_dir(source, target):
     if sys.platform == 'win32':
         subprocess.run(f'xcopy \"{source}\" \"{target}{os.path.sep}\" /E /Q')
```

Comment:
quick question about this: what is the purpose of calling `xcopy` instead of just using `shutil.copy`?
Are there any special properties that I am not aware of

Reply:
IIRC it was a lot faster